src/components/main/compose/msg/mod.rs

```diff
@@ -35,6 +36,7 @@ pub struct Props<'a> {
     first: bool,
     middle: bool,
     last: bool,
+    profile_picture: std::option::Option<std::string::String>,
```

Comment:
Just as an fyi, you can just do `Option<String>` since rust would do `std::option::Option<std::string::String>` at compiletime but not a blocker here just a matter of keeping things clean.

Reply:
That's good to know, thanks for the tip